lib/active_record_florder/configurable.rb

```diff
@@ -48,5 +52,9 @@ def min_position_delta
     def next_position_step
       self.class.next_position_step
     end
+
+    def move_populate_all
```

Comment:
At all I don't understand `populate` and `move_populate_all` it is not really intuitive. Without README I would guess it populate order of items with some values or something else related with populating.